openbas-front/src/admin/components/assets/Index.tsx

```diff
@@ -15,15 +17,17 @@ const useStyles = makeStyles()(() => ({ root: { flexGrow: 1 } }));
 
 const Index = () => {
   const { classes } = useStyles();
+  const ability = useContext(AbilityContext);
+
   return (
     <div className={classes.root}>
       <Suspense fallback={<Loader />}>
         <Routes>
-          <Route path="" element={<Navigate to="endpoints" replace={true} />} />
-          <Route path="endpoints" element={errorWrapper(Endpoints)()} />
-          <Route path="endpoints/:endpointId/*" element={errorWrapper(IndexEndpoint)()} />
-          <Route path="asset_groups" element={errorWrapper(AssetGroups)()} />
-          <Route path="security_platforms" element={errorWrapper(SecurityPlatforms)()} />
+          <Route path="" element={<Navigate to={ability.can('ACCESS', 'ASSETS') ? 'endpoints' : 'security_platforms'} replace={true} />} />
```

Comment:
I don't understand that.
What's happening if you don't have capa for Security Platforms and for endpoints ?